### PR TITLE
Login

### PR DESCRIPTION
**Pull request by titor-oopart**

### CODE REVIEW

File: src/main/java/core/utils/PivotalTrackerUtils.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        driver.close();
+    }
+
+    /**
+     * This method quit the browser after of the test.
+     */
+    public void quit() {
+        driver.quit();
+    }
+}
```
Comment (LimbertVargas) [inline src/main/java/core/utils/PivotalTrackerUtils.java]: please add a blank space

**Review by LimbertVargas — COMMENTED** [review]
please review the comment

File: src/main/java/core/utils/PivotalTrackerUtils.java (hunk 1 above)
Comment (titor-oopart) [reply src/main/java/core/utils/PivotalTrackerUtils.java]: done

**Review by LimbertVargas — APPROVED** [review]
It's good for me, good job

**Review by LimbertVargas — APPROVED** [review]

**Pull request merged**